### DISCUSSION TITLE
Extensive Talent review (+fantasy talents)

### DISCUSSION THREAD

**Discussion by ubertpendragon** [FakeFishGames/Barotrauma#9625] [Ideas and suggestions]
Here I'll talk about every single talent

# Captain

Overall, the captain talents are a money/xp printing machine, first and foremost. The talents aren't overly powerful in terms of raw stats.
There is an obvious unbalance for anyone that played Captain:

**Tree picks: Gunslinger vs Skipper vs Shipmaster**
Skipper gets you medals. Medals get you and your crew experience. You ALWAYS go skipper first. The faster you go skipper, the faster you'll snowball through every other talent.

**Gunslinger**:
-[Once Upon a Time on Europa](https://barotraumagame.com/wiki/Once_Upon_a_Time_on_Europa) is decent, opens the way to 100 weapon skill. Extra reputation gain is not noticeable since the game essentially gives too much reputation in general
-[Quickdraw](https://barotraumagame.com/wiki/Quickdraw) is a good concept but the numbers are bad. _Reduce the delay to 3 seconds and increase the damage to 50%_
-[Tumble](https://barotraumagame.com/wiki/Tumble): it's better than Quickdraw, but no one really likes ragdolling, and the duration is too short. _Increase the duration_
-[Drunken Sailor](https://barotraumagame.com/wiki/Drunken_Sailor) is decent, you just have to bear being drunk.
-[Sailor with No Name](https://barotraumagame.com/wiki/Sailor_with_No_Name): Good talent. Removing Reaper's Tax is good and cigars are fun for RP too
-[Bounty Hunter](https://barotraumagame.com/wiki/Bounty_Hunter) is a good T4 but suffers from the competion of the Skipper tree which considerably increases rewards by adding a 3rd mission.

**Skipper**
-[Evasive Maneuvers](https://barotraumagame.com/wiki/Evasive_Maneuvers): The 75 helm is overkill since you have 40 to 50 helm by default, so you're well over 100. Mission reward is nice. _Change the effect to either give 50 Helm to the entire crew instead_
-[Traveling Tradesman](https://barotraumagame.com/wiki/Traveling_Tradesman) is bugged and doesn't apply sales. It's a good talent otherwise
-[Trusted Captain](https://barotraumagame.com/wiki/Trusted_Captain) is where problems start. Medals giving XP to both the target and the captain makes rushing this talent the n°1 priority for captains. You can even buy medals from shops sometimes which lets you level even faster. _Remove the XP gain for the captain and replace it by a secondary effect (skill gain, temporary buff for the duration of the mission...)_  
-[Down with the Ship](https://barotraumagame.com/wiki/Down_with_the_Ship) isn't good by itself. Repairing a flooded sub is generally a downtime activity; you want repairs to be fast while the sub is sinking (7%+ water), not when it reaches as much as 50%+ water. It is even worse when the other option gives medals. You are actively losing XP by picking this _This is needs a massive buff to be worth taking. Apply the buff when inside any flooded room with >80% water level, increase repair speed, reduce duration_
-[Trucker](https://barotraumagame.com/wiki/Trucker) is decent money, but doubles down on medal xp. Even if you don't have missions, completing a beacon gives you up to 3 medals. _Only award a medal when 2 or more missions are completed_
-[Esteemed Captain](https://barotraumagame.com/wiki/Esteemed_Captain) gives even more medals/XP. _Reduce xp gain, reduce sub upgrade costs by 10%?_

**Shipmaster**
-[Commander](https://barotraumagame.com/wiki/Commander) is good for a T1
-[Maintenance Routine](https://barotraumagame.com/wiki/Maintenance_Routine) is decent, but not essential
-[Trust Building](https://barotraumagame.com/wiki/Trust_Building) is probably better than the alternative.
-[Camaraderie](https://barotraumagame.com/wiki/Camaraderie) suffers from the alternative being much better. 5 job requirement makes it unreliable on small subs and pub servers as well as not dying. You already get plenty of money and XP with skipper. _Give 4% extra money/xp per different job on the sub instead of a flat 20% +200mk to each individual crew member_
-[Inspire to Battle](https://barotraumagame.com/wiki/Inspire_to_Battle) is good, stun resistance is extremely valuable. 
-[All Hands on Deck](https://barotraumagame.com/wiki/All_Hands_on_Deck) is good.

Recap: 
Some gunslinger buffs to not make pistols a second-rate weapon, medals needs the nerfbat to prevent XP snowball, shipmaster talents are balanced.

# Security

Security gains some pretty massive stats from its talents, but overall they are pretty well made and offer a real choice

**Tree picks: Specialist vs Warden vs Soldier**
It's not an easy choice, which is good! All 3 trees offer something unique and strong at almost every tier. You probably won't pick T1 specialist perks without going deep in the tree. Bootcamp feels like an always must-pick for a 1st perk though since it gives XP.

**Specialist**
-[First Aid Training](https://barotraumagame.com/wiki/First_Aid_Training) is lacking compared to the raw stats of the alternative. Slightly better healing can be unnoticeable. _Add some bleeding resistance as well?_
-[Physical Conditioning](https://barotraumagame.com/wiki/Physical_Conditioning) increases your melee DPS by 25%, However you'll not notice 40% oxygen low resistance. _Increase oxygen low resistance to 200%?_
-[Navy Diver](https://barotraumagame.com/wiki/Navy_Diver) gives you damage reduction and that is the most important stat in the game, AND it gives you a cool diving suit on top of that. Consider the alternative...
-[Scavenger](https://barotraumagame.com/wiki/Scavenger) doesn't cut it, since you're not getting damage reduction. The loot is on average mediocre. Don't pick this over the alternative. _Increase quality of looted objects? Increase damage dealt when inside wrecks/caves?_
-[Buccaneer](https://barotraumagame.com/wiki/Buccaneer) is strong. Boarding axe is powerful, stun resistance is extremely good. 
-[Deep Sea Slayer](https://barotraumagame.com/wiki/Deep_Sea_Slayer) is also good. It will drop A LOT of alien materials against anything that leaves an inventory behind (leucocytes) so it might be a little too good at that. _Make alien materials rarer (10%), increase regular drops (alien blood...) instead?_
-[Dual Spec-Ops](https://barotraumagame.com/wiki/Dual_Spec-Ops): the dodge effect is nice, but the two free talent point means this is essentially an extra free talent which feels weird. _Give something else instead of 2 free talents?_

**Warden**
-[Enforcer](https://barotraumagame.com/wiki/Enforcer) is strong. +25% health is huge
-[Beat Cop](https://barotraumagame.com/wiki/Beat_Cop) is a funny, RP talent with no practical use in dry environments. It pairs well with Dual Spec Ops as you can stunlock the weaker enemies. _Not sure how to make it more reliable outside of water, and for what purpose_
-[Munitions Expertise](https://barotraumagame.com/wiki/Munitions_Expertise) is way too good for a T2, and generally better than Beat Cop. You get free ammunition which spares you precious resources, and you also get powerful explosive ammo recipe. _Move this to T3_
-[By the Book](https://barotraumagame.com/wiki/By_the_Book) is an excellent concept, but comes too late at T3 and doesn't give any stats, unlike its alternative. _Move this to T2_
-[Stonewall](https://barotraumagame.com/wiki/Stonewall) gives damage resistance, the most important stat. It is so much better than By the book. _Swap Munitions expertise and By the Book_
-[Implacable](https://barotraumagame.com/wiki/Implacable) is strong and autoshotgun is fun.

**Soldier**
-[Bootcamp](https://barotraumagame.com/wiki/Bootcamp) gives +35% XP making it the must-pick for all T1s, and like all other XP talents snowballs throughout the campaign. _Gain +10%xp and +1 to all skills instead?_
-[Gunsmith](https://barotraumagame.com/wiki/Gunsmith) is good. Bandolier is strong and so is extra weapon quality.
-[Warrior Poet](https://barotraumagame.com/wiki/Warrior_Poet) gives you immense amounts of cash, and apparently it is getting canned. Money isn't necessarily limiting in a campaign so Gunsmith still feels better
-[Expert Commando](https://barotraumagame.com/wiki/Expert_Commando) is a little akward to use and not many people know it works underwater. Less shotgun spread is appreciable but other weapos don't have much spread. _It needs some love_
-[Tandem Fire](https://barotraumagame.com/wiki/Tandem_Fire) passively increase the DPS of two of your sub guns. You conserve ammunition and kill enemies faster; this is much more important than Expert Commando
-[Warlord](https://barotraumagame.com/wiki/Warlord): Assault rifle is great but you will not notice the 5% crit chance. 

Recap: 
Overall pretty good. Some minor changes and a few not-so-great talents

# Medical doctor

The medic has at the same time both some of the strongest, game changing talents, and talent trees you will not notice at all.

**Tree picks: Xenologist vs Medic vs Scientist**
Medic is useful, all the time, and is the gateway to every other talents with endocrine booster and therefore feels like a must-pick for the first tree. Xenologist gives insane amounts of rewards... but you also can play an entire campaign without ever doing ruins. Same goes for Scientist, which is on the weaker side, as you can completely avoid the Genetic system.

**Xenologist**
-[Researcher's Intuition](https://barotraumagame.com/wiki/Researcher%27s_Intuition) is, like most Xenologist talents, completely useless early game as Ruins only start spawning naturally from biome 3 onwards. The 50% increased rewards however, are pretty extreme and need a nerf. _See below for why_
-[Curiosity](https://barotraumagame.com/wiki/Curiosity) XP gain is nice, but once again only works starting on 3rd biome... _Give xp for interacting with alien items in general (example: deconstructing Alien Blood, hatching eggs...)_
-[Double Duty](https://barotraumagame.com/wiki/Double_Duty) gives damage reduction, the most important stat.
-[Alien Hoarder](https://barotraumagame.com/wiki/Alien_Hoarder) is not good. Extremely niche, requires you to carry up to 5 artifacts (you want to deconstruct them asap), and boosts a weapon which ammunition can't be refilled... _Unlock Alien Power Cell refill recipe using Fulgurium + Increase damage for melee too, reduce the amount of artifacts needed_
-[Reverse Engineer](https://barotraumagame.com/wiki/Reverse_Engineer) double dips on alien resources, giving you insane amounts of alien materials and free skills. Again needs to not just interact with ruins items. _Remove double output, allow combining 5x Alien Blood into Unidentified Genetic Material ala Scrap Savant?_
-[Atmos Machine](https://barotraumagame.com/wiki/Atmos_Machine) triple dips on alien resources. ARTIFACTS CAN DECONSTRUCT INTO ARTIFACTS, into even more resources. Most importantly, it's a boring T4 on a boring talent tree. _Give it a fun effect, like a 10000 range discharge coil, or repairs ALL electronics after deconstructing an artifact_

**Medic**
-[Field Medic](https://barotraumagame.com/wiki/Field_Medic) is good because extra health
-[Fireman's Carry](https://barotraumagame.com/wiki/Fireman%27s_Carry) is very fun, very useful.
-[Mechanics Course](https://barotraumagame.com/wiki/Mechanics_Course) is also very useful since headset is a free slot
-[Bedside Manner](https://barotraumagame.com/wiki/Bedside_Manner) is _nice_ as medicine is constantly applied on the entire crew... but unless you're using books it isn't obvious how skill gain works.
-[Combat Medic](https://barotraumagame.com/wiki/Combat_Medic) is the stronger pick as it gives you stats, and combat stimulant being very one of the most powerful medicines in the game. 
-[Don't Die on Me!](https://barotraumagame.com/wiki/Don%27t_Die_on_Me!) gives Endocrine booster... which is a must-have and makes picking the Medic tree a priority as you can accumulate talents as fast as possible. 1 Endocrine booster is essentially 1 level! _On the fence about Endocrine boosters. They're really fun but also too powerful... Limit to 1 booster per character, or 1 talent per class?_

**Scientist**
-[Dr. Submarine](https://barotraumagame.com/wiki/Dr._Submarine) is weak. It saves you a tiny bit of meds (on yourself) and some meaningless opiate resistance... _Gain Hyperactivity from applying meds?_
-[Gene Therapist](https://barotraumagame.com/wiki/Gene_Therapist) also saves you meds, but better by making meds stronger in general, it is the better talent.
-[Research and Development](https://barotraumagame.com/wiki/Research_and_Development) is good thanks to the XP. 
-[Research Grant](https://barotraumagame.com/wiki/Research_Grant) is also good, gives some spare cash and pressure stabilizer allows using powerful body armour outside the sub... but it is also very expensive and doesn't last long. _Reduce recipe cost to 1x Oxygenite, increase yield? 100 seconds is very short for EVA_
-[Gene Harvester](https://barotraumagame.com/wiki/Gene_Harvester) is a great concept, and makes genes available anywhere at any time.
-[Genetic Stability](https://barotraumagame.com/wiki/Genetic_Stability) is good for minmaxing genes; but relies on finding genes via conventional methods unlike harvester. _Increase Genetic material drop chance from Thalamus organs?_
-[Advanced Splicing](https://barotraumagame.com/wiki/Advanced_Splicing) is mostly only worth it for the Double gene splicer. The XP gain is not noticeable for a T4, and the extra medical skill most has minor benefits. _Add some Husk/Paralysis/Poison resistance? Anything?_

Recap:
Xenologist and Scientist trees rely on aspects of the game (ruins, genes) you can completely ignore in a playthrough. Xenologist gives way too many resources, triple-dips the loot tables. Endocrine boosters are absurdly good. Genes need to be more accessible in general for Scientist to work.

# Engineer

Engineer talents are overall pretty powerful, sometimes for the wrong reasons, while others are useless.

**Tree picks: Weapons Engineer vs Electrician vs Physicist**
Electrician is never a priority pick. It's a QoL tree while the two others give you huge offensive/defensive recipes.

**Weapons Engineer**
-[Military Applications](https://barotraumagame.com/wiki/Military_Applications) gives you upgraded coilgun ammo as early as T1, on top of a substantial weapons bonus. It's a little too good for such an early investment. _Move to Tier 2 (replaces Strengthened Alloys), reduce weapons bonus to 10_
-[Arms Dealer](https://barotraumagame.com/wiki/Arms_Dealer) prints Magnesium and Steel on the fly from any mission, and gives you access to more powerful SMG ammo. Damage is very important, more so than repair speed. _Only give resources after Beacon/Abandoned outpost/Cargo missions_
-[Strengthened Alloys](https://barotraumagame.com/wiki/Strengthened_Alloys) is nice, but you won't pick repair speed over more SMG damage. The knife even does less bleeding. _Move to T1, gain the Depleted fuel recipe (replaces Military Application, reduce mechanical bonus to 10_
-[Castle Doctrine](https://barotraumagame.com/wiki/Castle_Doctrine) isn't great. Dirty bombs are rarely used. Extra damage inside the sub is situational. _Gain 80% explosive damage resistance inside the sub?_
-[Expanded Research](https://barotraumagame.com/wiki/Expanded_Research) is 600 free XP per mission, thanks to the huge amount of fuel rods given by Nuclear option (see below). _Nerf fuel output, Deconstructing depleted fuel gives Lead?_
-[Nuclear Option](https://barotraumagame.com/wiki/Nuclear_Option) essentially gives unlimited fuel rods / depleted fuel. You can even redock to a station without completing a mission to get triple output again. The rapid fissile accelerator is a fun weapon, but the triple fuel rod output, combined with the higher fuel rod quality, combined with Arms dealer means you are printing free ammo for BOTH RFA and SMG as well as experience, and trivializes fuel consumption. It's good enough without it. _Remove the triple fuel rod output_

**Electrician**
-[Just a Scratch](https://barotraumagame.com/wiki/Just_a_Scratch) is weak. If you stay under 50% you have a death wish. _Increase health cap to 80%_
-[Minor in Mechanics](https://barotraumagame.com/wiki/Minor_in_Mechanics) is QoL, but you won't notice it exists compared to, say, Depleted Coilgun ammunition. 
-[In the Flow](https://barotraumagame.com/wiki/In_the_Flow) is good, helps repairing the aftermath of overvolting or a sunken sub.
-[Melodic Respite](https://barotraumagame.com/wiki/Melodic_Respite) is also good.
-[Electrochemist](https://barotraumagame.com/wiki/Electrochemist) is strong, damage reduction is the most important stat, although monsters are mostly immuned to stun. Battery consumption in general never matters so extra battery quality is only a factor for stun batons. _Increase battery power consumption_
-[Optimized Power-Flow](https://barotraumagame.com/wiki/Optimized_Power-Flow) doesn't matter, because turret power consumption is way too low in general to matter. You can rack up a ton of electrical points easily, but it only really results in fast repairs. _Increase turret power consumption_
-[All-Seeing Eye](https://barotraumagame.com/wiki/All-Seeing_Eye) makes all of your previous choices irrelevant, and in theory gives you 3 free talents but the entire tree is mediocre. Thermal goggles are nice though. _Buff other talents to buff this, also needs a T4 alternative that is good enough_

**Physicist**
-[Remote Monitor](https://barotraumagame.com/wiki/Remote_Monitor) is only good for the fuel rod quality. You never touch the reactor (it's always either auto or ARCs). _Entirely relies on making manual reactor a thing_
-[Co-Author](https://barotraumagame.com/wiki/Co-Author) is nice, some xp, some skills for the whole crew. 
-[Hazardous Materials](https://barotraumagame.com/wiki/Hazardous_Materials) gives you some free alien materials, which is good, and the cool but mighty useless volatile fulgurium rod. Fuel is already a non-issue, the rod is hazardous for the crew, and can't even be used as a weapon (even in a depth charge). _A reactor/fuel rework and rebalancing is required for volatile rods to have a purpose_
-[College Athletics](https://barotraumagame.com/wiki/College_Athletics) gives MASSIVE stats and the extremely useful cargo scooter. So many stats rolled into one talent feels a little too good,  and it's hard to pass up on the scooter... _15% max health bonus at most_
-[Polymath](https://barotraumagame.com/wiki/Polymath) is in theory the better option if you never leave the sub, gets you more talents faster, but you don't have a scooter for the whole crew... _This talent needs some love, but how?_
-[Ph.D in Nuclear Physics](https://barotraumagame.com/wiki/Ph.D_in_Nuclear_Physics) makes Crowbar melee build fun, repairing items to 140% is appreciated... but lets face it PUCS is overpowered. It has all the stats of the combat suit + Burn/Husk/Radiation near-immunity + Auto-injector for good measure. _Nerf PUCS combat damage resistances (gunshot/bite/bleeding/trauma/lacerations)_

Recap:
Cheap, powerful ammunition with unlimited fuel on one hand, one of the best utlity items and suit on the other, and in the middle a mediocre set of talents bound by external factors (reactors, power). 

# Mechanic

Mechanic talents, unlike the engineer, shine for the right reasons: the utility they bring cement the class as a support role.

**Tree picks: Machinist vs Technician vs Pioneer**
They're all good. Technician feels a little weaker because it doesn't have tinkering or the scrap economy. Pioneer is prefered due to xp gain from scrap.

**Machinist**
-[Tinkerer](https://barotraumagame.com/wiki/Tinkerer) is awesome. Tinkering is always useful.
-[Mass Production](https://barotraumagame.com/wiki/Mass_Production) is generally prefered to the alternative, because fabricators/deconstructors can be crowded. Not a game-changing talent but very nice QoL
-[Quickfix](https://barotraumagame.com/wiki/Quickfix) is pretty good since the recipe change. XP, faster repairs and no skill checks.
-[Hull Sealer](https://barotraumagame.com/wiki/Hull_Sealer): Fixfoams are actually very cheap and powerful; fixing the entire hull of a room instantly at the cost of some steel and sodium, and can even be automated via detonators. _Recipe probably needs to be changed to be a little more expensive; use raw Graphite?_
-[Overclocking](https://barotraumagame.com/wiki/Overclocking) is nice, but it is no Fixfoam...
-[Let It Drain](https://barotraumagame.com/wiki/Let_It_Drain) has an interesting "power" effect, but most importantly it gives you portable... black holes. The pumps have a draining power of 1000 which is frankly insane (10x that of a small pump). They can even be automated via logic. _Reduce pumping power to 300 and reduce battery power consumption by half so they are used as support pumps_

**Technician**
-[Aggressive Engineering](https://barotraumagame.com/wiki/Aggressive_Engineering): Powerful if you can reliably stunlock creatures with Dementonite tools.
-[Safety First](https://barotraumagame.com/wiki/Safety_First): Safety harness is pretty useful, especially on bots. It's generally more useful than Akimbo wrenches.
-[Hull Fixer](https://barotraumagame.com/wiki/Hull_Fixer) is great, helps everyone on the sub as you make better welding tools. 
-[Crisis Management](https://barotraumagame.com/wiki/Crisis_Management) is not good for the same reasons as Down with the Ship. When 5 players are repairing, one player repairing 75% faster is not noticeable.  _Apply the repair speed bonus to all nearby characters so the repairs are actually fast_ 
-[Pyromaniac](https://barotraumagame.com/wiki/Pyromaniac) gives damage resistance, which is great, and increases Laser guns or Steam canon damage.
-[Artisan Smith](https://barotraumagame.com/wiki/Artisan_Smith) is good.

**Pioneer**
-[Scrounger](https://barotraumagame.com/wiki/Scrounger) is a good talent; gives free resources and XP, although too much as you can find reliably scrap in supply cabinets and smaller containers. _Reduce scrap finding chance to 20%_
-[Demolitions Expert](https://barotraumagame.com/wiki/Demolitions_Expert): Explosive quality is generally useless because it doesn't affect all types of explosive ammunition. It slightly increases the damage of loaded railgun shells, but explosives are generally impractical.  _Increase explosive damage/radius so it can work with explosive harpoons or explosive coilgun ammo_
-[Miner](https://barotraumagame.com/wiki/Miner) gives way too many extra materials from mining, when resource scarcity is already hardly a factor, on top of the extra scrap from other talents. _Reduce double output chance to 20%_
-[Canned Heat](https://barotraumagame.com/wiki/Canned_Heat) is good. Free XP and better tanks is always useful.
-[Elbow Grease](https://barotraumagame.com/wiki/Elbow_Grease) is overpowered. You can easily get over 100 mechanical, up to 230 with talents alone which will give you 92+% health and melee damage, on top of the benefits of instantly repairing items. _20% Mechanical into health/damage at most_
-[Scrap Savant](https://barotraumagame.com/wiki/Scrap_Savant): Constructing alien materials on the fly is great, but it gives way too much scrap, and double dips on finding scrap in wrecks. It's not uncommon to leave a wreck with 20 scrap, which ensures a lot of XP and guaranteed materials of your choice. _Reduce scrap finding chance to 20%_

Recap:
Pioneer tends to give too many resources, but all trees are well-designed in general. Tinker is awesome. 

# Assistant

Assistant talents are a mixed bag. Some are fun and interesting, like clown talents, while the Jack skill tree is... boring?

**Tree picks: Jack of All Trades vs Grayshirt vs Clown**
Jack of all trades has the potential to snowball, generating XP and skill points for the entire crew. Clown has powerful talents, while Grayshirt is interesting but not as good.

**Jack of all trades**
-[Apprenticeship](https://barotraumagame.com/wiki/Apprenticeship) is good as it gives health and free skill points passively, which is important for the entire tree
-[Multitasker](https://barotraumagame.com/wiki/Multitasker) is also good but less effective on smaller crews. 
-[Stand and Deliver](https://barotraumagame.com/wiki/Stand_and_Deliver) is great. Free skills
-[Ace of All Trades](https://barotraumagame.com/wiki/Ace_of_All_Trades) is good and so is the alternative
-[Rich in Knowledge](https://barotraumagame.com/wiki/Rich_in_Knowledge) is good and so is the alternative
-[Olympian](https://barotraumagame.com/wiki/Olympian) only works for faster repairs and faster driving. Extra medical and weapons do nothing. _If you have 200 skill, allow crafting Masterwork quality?_

**Grayshirt**
-[Playing Catchup](https://barotraumagame.com/wiki/Playing_Catchup) competes with stats, and is useless if you start the game as an assistant. _Increase XP gain to 200%?_
-[Skedaddle](https://barotraumagame.com/wiki/Skedaddle): Stun resistance and speed are good.
-[Crew Layabout](https://barotraumagame.com/wiki/Crew_Layabout) is good if you idle/RP a lot and don't care about dying
-[Survival Package](https://barotraumagame.com/wiki/Survival_Package): Ragdoll doesn't seem to work at all. The talent prints about 300mk of medical supplies per mission. _Give medical items on respawn instead to match the theme of the tree_
-[Insurance Policy](https://barotraumagame.com/wiki/Insurance_Policy) removes Reaper's Tax although it doesn't say it, allowing you to take risks and die in missions without penalty to health. It's a great money-maker too.
-[Still Kicking](https://barotraumagame.com/wiki/Still_Kicking) is also great for taking risks
-[Graduation Ceremony](https://barotraumagame.com/wiki/Graduation_Ceremony): The talent incentivizes you to reroll your character... but it also only gives more health to the assistant's current character, so which is it? You have to commit to 4 tiers of assistant that don't do anything to reroll into a character that actually matters. _At least gain one extra level or 5000 extra XP when rerolling so you get free levels_.

**Clown** _so apparently the clown tree is going away from assistant_
-[Enroll Into Clown College](https://barotraumagame.com/wiki/Enroll_Into_Clown_College) gives damage resistance, the most important stat of all. Skill gain is minimal
-[Inspiring Tunes](https://barotraumagame.com/wiki/Inspiring_Tunes) is great for taking care of maintenance, but you can't go outside as a clown. Which is why... 
-[Water Prankster](https://barotraumagame.com/wiki/Water_Prankster) is what most players will pick. Underwater clown is absolutely unique AND powerful as you can equip body armor underwater, It is a staple of the tree and defines clown as a whole.
-[Psycho Clown](https://barotraumagame.com/wiki/Psycho_Clown) is very powerful and getting psychosis is as easy as injecting alien blood.
-[Slapstick Expert](https://barotraumagame.com/wiki/Slapstick_Expert) is great, allows you to go limp before geting hit, reducing damage and stunning the attacker
-[True Potential](https://barotraumagame.com/wiki/True_Potential) is completely broken. It reduces damage taken, which is on top of the reistance from clown college, on top of body armor, and singlehandedly saves HUGE amounts of medical supplies by healing for free. All of the damage suffered from repairs does not require bandages. You can heal faster than bleeding, you can heal everything up to 100%. _Remove the extra damage resistance, limit the healing to a cap of 80% of max health so you still require medical items to heal the crew_

Recap: Clowns are overpowered, Jack of all trades is pretty uninteresting, and Greyshirt has the most unique mechanics but its T4 doesn't make sense.

**Comment by ubertpendragon** [comment]
And now a list of talent ideas to fill the gap I spent entirely too much time on:

# Captain

**True Grit (Gunslinger T1)**
Revolver rounds in outposts cost 50% less
Shooting your crew gives them 30% increased repair, movement speed, and melee attack speed

**Desperado (Gunslinger T4)**
Increase Revolver fire rate by 200%
Enemies killed drop 3 revolver bullets.

**Team builder (Skipper T2)**
Hiring bots costs 20% less
Gain 100xp for each uninjured bot when completing a mission

**Corsair (skipper T4)**
When you complete a pirate elimination, gain two Coalition Commendations.
Outposts always offer pirate missions.
Gain a bonus of 20 to weapons

**Second in command (Shipmaster T1)**
Whenever you give an order to another character, they gain 50 helm and all your current buffs

**Not today(Shipmaster T4)**
Whenever you give an order to an unconscious character, revive him instantly to 50% health and apply 100 vigor
Works once per mission

----------
# Security

**Dynamite fishing (Specialist T4)**
Increase the stun of explosions when underwater by 100%
Recover from stun 50% faster underwater
Gain 30% explosive damage resistance

**Big game hunter (Warden T4)**
Increase the ragdoll force of shotguns by 150%
Decrease weapon spread by 50%
If there's only one shell left, increase shotgun damage by 100%

**Basic training (Soldier T1)**
Ragdoll for 1 second to get a 5s exercise buff.
Exercise gives you 15% increased damage, 10% damage reduction 20% less spread, 20% move speed and 20% melee weapon speed for up to 120s

**Pocket Artillery (Soldier T4)**
Unlocks recipe: Raildar gun, Raildar shell
The Raildar gun is a heavy weapon with an integrated sonar display. Fires pocket railgun shells.
Increases the view distance on submarine guns by 15%

-------

# Medical Doctor

**Deadly Consultation (Xenologist T1)**
Gain 50 xp for each enemy that dies inside your sub, and a 15% chance of dropping a Genetic material.

**Call of the Deep (Xenologist T4)**
Unlocks recipe: Psychosis artifact
Crewmembers gain +30% movement speed, 100% vigor and gain pressure resistance for 60 seconds if they have over 50% psychosis

**Botanist (Medic T1)**
Plant-based products are twice as effective (pomgrenade, raptor bane, tobacco)
You get 1 more item from deconstructing Aquatic Poppy, Elastin Plant, Fiber Plant, Sea Yeast and Slime Bacteria

**Throat medicine (Medic T4)**
Injection of husk eggs rapidly heal to full health. 
Vomiting gives 80% damage resistance for 3 seconds and deals damage to enemies.

**Dissector (Scientist T4)**
If you kill and dismember a friendly crewmember, gain 4 blood bags, +5 to all skills and one of its Tier 1 talents.

------

# Engineer

**Brilliance (Weapons Engineer T1)**
Unlock reciple: Depleted fuel Railgun Shell.
Picking up used fulgurium rods turns them into depleted fuel.
You emit light and heal wounds slowly if you have radiation sickness

**Inert Carbon God(Weapons Engineer T4)**
Unlocks reciple: Enriched Pulse Laser Ammunition
Fuel rods loaded in railgun shells become unstable and explode on impact
If you hold a fuel rod underwater, enemies won't attack you


**Electro-Tactician(Electrician T4)**
Charging an empty supercapacitor to full gives all connected turrets +25% attack speed and damage for 10 seconds
You can heal yourself by interacting with supercapacitors, draining their energy
Stun batons deal 1000% more damage when powered

**Alchemist(Physicist T1)**
When you deconstruct Fulgurium, gain +2 to all skills and another alien material
+100% fabricator speed

**The Final Meltdown (Physicist T4)**
If you meltdown the reactor, create a massive explosion that deals 1000 damage to monsters in a large radius.
Gain 50% explosion burn and damage resistance when the reactor is critical.
+35% repair speed

-------

# Mechanic

**Machine Learning (Machinist T1)**
Whenever you deconstruct a talent item, unlock its recipe
-10 to all skills

**Pit stop (Machinist T4)**
For each location visited that is NOT an outpost, your crew receives 300xp, 5 mechanical skill points 3 repair packs upon docking on the next outpost.
Upon visiting a beacon, you receive one random submarine upgrade

**Shaken, not stirred (Technican T2)**
Crouching up and down rapidly for 10 seconds while holding a fuel tank will turn it into a throwable grenade.
Breathing welding fuel gives you Vigor and Health regeneration for 20 seconds
Gain 2 Ethanol for each completed mission.

**Gears of war (Technician T4)**
When you break a door, deal massive damage on the opposite side
+15% melee damage, explosion damage, burn damage, blunt damage, explosion radius, stun, stun resistance
+100% fabrication speed for ammunition

**Piecemaker(Pioneer T1)**
Gain 2 scrap whenever you deconstruct a weapon
When you deconstruct scrap, gain +1 to mechanical and medical
Weld doors 300% faster

**Savage Salvage (Pioneer T4)**
You can load scrap into a shotgun. Deals 3x shotgun damage with +50% spread.
Drops resources on impact.
50% chance to find scrap inside pirate ships and abandoned outposts containers.

------

# Assistant

**Brain-eating Amoeba (Jack of all trades T2)**
Lose 5 of all stats every time you complete a mission.
Every time you lose stats this way,gain +10% melee damage and damage resistance, and take 2 internal damage each minute
Your skills are reset upon death

**Apprenticeship (Jack of all trades T4)**
Whenever you are ordered by a crewmate, gain all of its tier 1 talents.
Gain +20 of their class skill

**Brain chip protocol (Grayshirt T4)**
Whenever you are eaten, take control of the creature that eats you.
Gain skills 100% faster.

**Trippy (Clown T1)**
Unlock reciple: Banana peel
Characters that walk on banana peels trip, and you gain 50xp (up to 200 per mission)
Whenever you play an instrument, nearby crewmates gain Drunk and you gain up to 50% move speed

**Honky Bonk (Clown T4)**
Honking your horn scares monsters away from you. You have a 0.75% to explode from honking your horn.
Toy Hammer can revive unconscious teamates with a combat stimulant bonus.
Gain pressure and low O2 immunity.

**Reply by mLuby** [reply]
Most of those suggestions are _**pure gold**_. 👏 A few counter-suggestions, either replacement for the quoted part or the entire thing:

> Captain Gunslinger T4 "Desperado" Enemies killed drop 3 revolver bullets

Instead: "Trigger Fan" If you're only holding one revolver, it [shoot]s faster and fully auto (where normally it _ought_ to fire once per [shoot]). Also enable a craftable "moon clip" item that carries 36 revolver bullets for ease of reloading. Why: the 8-stacks clog inventory and your Gunslinger T1 suggestion will make rounds plentiful enough already.

> Security Soldier T4 "Pocket Artillery"

Instead: "Ammo Conservationist" when attempting to fire a turret with an _empty_ ammo box, the shot consumes no power and does 200% damage. Can only happen once every 10 seconds. (Empty railgun fires a canister shot, so there's nothing to pick up.)

> Mechanic Technician T2 "Shaken, not stirred"

Instead: "Shaken, no stirred" [crouch]ing and [aim]ing for 10 seconds while holding ethanol and a bandage turns them into a Volatile Incendiary Grenade. Being drunk gives you +20% resistance to all damage types. Gain 1 Ethanol and 1 Bandage each round you complete at least one mission.

> Mechanic Pioneer T2 "[Demolitions Expert](https://barotraumagame.com/wiki/Demolitions_Expert)"

Additionally: unlocks a Smoke Grenade blueprint that creates a cloud of smoke that quickly obscures the room. [Useful for melee, especially Buccaneers, or anyone using Thermal Goggles.]

> Mechanic Pioneer T4 "Savage Salvage" 50% chance to find scrap inside pirate ships and abandoned outposts containers.

Instead: _You find the machines you repair came with too many screws_: once per round per mechanical machine you repair to full, you find 1 scrap.

**Comment by ubertpendragon** [comment]
Since the talent rework is underway I figured out, might as well post this.

Most of the talents have been rebalanced according to the above post, some custom talents have been added. 
There's a changelog for both in the folder
It's a WIP

[Talentsrework.zip](https://github.com/Regalis11/Barotrauma/files/9717561/Talentsrework.zip)
